src/BagReader.test.js

```diff
@@ -12,7 +12,7 @@ import BagReader from "./BagReader";
 function int64Buffer(number: number) {
   const buff = Buffer.alloc(8);
```

Comment:
Can write

```javascript
const u = new Uint8Array(8);
new DataView(u.buffer).setBigInt64(0, BigInt(number), true);
return u;
```
if we want to drop the `Buffer` use here. No preference from me.

Reply:
yeah so this comes back to my other question about int53. Is it doing something special as to represent 64bits as 53? If so, this replacement isn't exactly the same, right?